From a0c6792733ec776c7cdb146c7c579c65004072ee Mon Sep 17 00:00:00 2001
From: "djm@openbsd.org" <djm@openbsd.org>
Date: Tue, 7 Jul 2026 02:17:18 +0000
Subject: [PATCH 07/12] upstream: unused variables

OpenBSD-Commit-ID: 03fc22fb427b7547ee7844907cf3257bce7fdc3c
(cherry picked from commit c10e04980300eab089f71aaada8b28fd878ee729)
[CVE-2026-60000 part 2]
---
 auth2-gss.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/auth2-gss.c b/auth2-gss.c
index 72be8ba68b6f..a3ed8944786d 100644
--- a/auth2-gss.c
+++ b/auth2-gss.c
@@ -215,8 +215,6 @@ input_gssapi_errtok(int type, uint32_t plen, struct ssh *ssh)
 {
 	Authctxt *authctxt = ssh->authctxt;
 	int r;
-	u_char *p;
-	size_t len;
 
 	if (authctxt == NULL)
 		fatal("No authentication or GSSAPI context");
-- 
2.55.0.dirty

